src/components/MultiHopTrade/components/TradeInput/components/Claims/Claims.tsx

```diff
@@ -0,0 +1,54 @@
+import { Box, CardBody } from '@chakra-ui/react'
+import { useCallback } from 'react'
+import { fox } from 'test/mocks/assets'
+import { ClaimRow } from 'components/ClaimRow/ClaimRow'
+import { ClaimStatus } from 'components/ClaimRow/types'
+import { Text } from 'components/Text'
+
+export const Claims = () => {
+  const tooltipText = 'tooltip text'
+  const actionText = 'Bridge via Arbitrum'
+  const statusText = 'status text'
+  const stakingAsset = fox
+  const handleClaimClick = useCallback(() => {}, [])
+  return (
```

Comment:
It's apparent after looking at this code for a moment, but maybe add a comment here just saying that this is placeholder code.

Reply:
Added a TODO to make this clearer